stubs/stripe/stripe/api_resources/list_object.pyi

```diff
@@ -22,7 +22,7 @@ class ListObject(StripeObject):
     def __iter__(self): ...
     def __len__(self): ...
     def __reversed__(self): ...
-    def auto_paging_iter(self) -> Generator[Any, None, None]: ...
+    def auto_paging_iter(self) -> Iterator[Any]: ...
     @classmethod
     def empty_list(cls, api_key: Any | None = ..., stripe_version: Any | None = ..., stripe_account: Any | None = ...): ...
     @property
```

Comment:
I can't add a suggestion but the `is_empty` property seems like it returns a `bool`, may as well update it while we're here